File: CF/Algolympics2020/P.py
def gcd(a,b):
    if b==0: return a
    else: return gcd(b, a%b)

t = int(input())

for i in range(t):
    k,b = [int(x) for x in input().split()]
    #k and b have a common factor
    if gcd(k,b-1)==1: print("COOL")
    else: print("NOT COOL")

File: CF/Algolympics2020/P.cpp
#include <bits/stdc++.h>
using namespace std;

int gcd(int a, int b) {
	if (b==0)
}

int main() {
	int t; cin>>t;

	while (t--) {
		int k,b; cin>>k>>b;
		
		b-=1;
		bool cool=true;
		for (int j=2; j<max(k,b); j++) {
			if (b%j==0 and k%j==0) {
				cool=false;
				break;
			}
			if (j>2) j++;
		}

		if (cool) cout << "COOL" << endl;
		else cout << "NOT COOL" << endl;
	}

	return 0;
}
